src/ContactPage/Contact.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import "./Contact.scss";
+
+class Contact extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      form: {
+        fullName: "",
+        _replyto: "",
+        subject: "",
+        message: ""
+      },
+      errors: {
+        fullName: "",
+        _replyto: "",
+        subject: "",
+        message: ""
+      }
+    };
+  }
+
+  handleChange = event => {
+    event.preventDefault();
+    const validEmailRegex = RegExp(
+      /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i
+    );
+
+    const { name, value } = event.target;
+    let errors = this.state.errors;
+
+    switch (name) {
+      case "fullName":
+        errors.fullName = value.length < 5 ? "Please enter your full name" : "";
+        break;
+      case "_replyto":
+        errors._replyto = validEmailRegex.test(value)
+          ? ""
+          : "Email is not valid";
+        break;
+      case "subject":
+        errors.subject = value.length < 3 ? "Please enter a subject" : "";
+        break;
+      case "message":
+        errors.message = value.length < 5 ? "Please enter a message" : "";
+      default:
+        break;
+    }
+    let form = { ...this.state.form };
+    form[event.target.name] = event.target.value;
+    this.setState({ form: form });
+  };
+
+  formCanBeSubmitted = () => {
+    const errors = Object.values(this.state.errors);
+    const form = Object.values(this.state.form);
+
+    function isFilledIn(element) {
```

Comment:
I'd make this arrow function to be consistent (also good practice to use newer syntax)

Same for below